.github/actions/secure-download-folder/action.yml

```
@@ -0,0 +1,91 @@
+name: "Secure folder download"
+description: "Download a folder and verify its SHA256"
```

```suggestion
description: "Download a folder uploaded by secure-upload-folder and verify its SHA256"
```

because it relies on the internals of secure-upload-folder (the hardcoded naming of folder.tgz)

---

done, thanks